data/kant_aufklaerung_1784/url-sources.txt

```diff
@@ -0,0 +1,2 @@
+https://github.com/OCR-D/assets/raw/master/data/kant_aufklaerung_1784/kant_aufklaerung_1784_0017.tif data/kant_aufklaerung_1784_0017.tif
+https://github.com/OCR-D/assets/raw/master/data/kant_aufklaerung_1784/kant_aufklaerung_1784_0020.tif data/kant_aufklaerung_1784_0020.tif
```

Comment:
Are these URLs correct?
The files will be moved during this pull request to https://github.com/OCR-D/assets/raw/master/data/kant_aufklaerung_1784/data/kant_aufk....., won't they?

Reply:
Deleted url-sources